### DISCUSSION TITLE
How to request an invite to community.infinite.red ?

### DISCUSSION THREAD

**Discussion by sondogF123** [infinitered/ignite#2105] [Q&A] [answered]
Trying to access community.infinite.red and receiving
<img width="671" alt="image" src="https://user-images.githubusercontent.com/46195945/187920272-f6cf6080-bd74-400e-b7bf-ed3b9e8dd285.png">

cant seem to find anywhere in github or infinite red website to request an invite.
apologies in advance for this non-technical discussion question.

**Comment by jamonholmgren** [comment, marked as answer]
Sorry about that! I fixed it and it should be working now.
